search.go

```diff
@@ -116,6 +135,7 @@ func getPackageOutput(ctx context.Context, packageList packages.Packages) ([]byt
 
 	// Packages need to be sorted to be always outputted in the same order
 	sort.Sort(packageList)
+	// FIXME ensure unique packageName:packageVersion
```

Comment:
This should be  already ensured by `packages.Join()`.

Reply:
Fixed